server/accounts.go

```diff
@@ -51,10 +51,11 @@ type Account struct {
 	imports    importMap
 	exports    exportMap
 	limits
-	nae     int32
-	pruning bool
-	expired bool
-	srv     *Server // server this account is registered with (possibly nil)
+	nae         int32
+	pruning     bool
+	expired     bool
+	srv         *Server // server this account is registered with (possibly nil)
+	signingKeys map[string]bool
```

Comment:
Do we need a map here? If so make struct{} takes up no space.

Reply:
changed to a slice